owner-onboarding-server/src/main.rs

```diff
@@ -19,15 +24,19 @@ use tokio::signal::unix::{signal, SignalKind};
 use warp::Filter;
 
 use fdo_data_formats::{
-    enhanced_types::X5Bag, ownershipvoucher::OwnershipVoucher, publickey::PublicKey, types::Guid,
+    constants::TransportProtocol,
+    enhanced_types::X5Bag,
+    ownershipvoucher::OwnershipVoucher,
+    publickey::PublicKey,
+    types::{Guid, TO2AddressEntry},
 };
 use fdo_store::{Store, StoreDriver};
 use fdo_util::servers::{settings_for, OwnershipVoucherStoreMetadataKey};
 
 mod handlers;
 mod serviceinfo;
 
-struct OwnerServiceUD {
+pub struct OwnerServiceUD {
```

Comment:
Let's make this a `pub(crate)`?

Reply:
this is wrong, I'll drop it - and the handler for report to rendezvous is still work in progress (pending some more learning around warp filters :))

Reply:
Ah, okay.